airflow/auth/managers/models/base_user.py

```diff
@@ -23,10 +23,6 @@
 class BaseUser:
     """User model interface."""
 
-    @property
-    def is_active(self) -> bool:
```

Comment:
Do we need a newsfragment for this? Technically it's a breaking change.

Reply:
You're right. I added one